hw/bsp/brtmm90x/boards/mm900evxb/board.h

```diff
@@ -27,21 +27,55 @@
 #ifndef BOARD_H_
 #define BOARD_H_
 
-// Note: This definition file covers all MM900EV1B, MM900EV2B, and MM900EV3B boards.
+// Note: This definition file covers all MM900EV1B, MM900EV2B, MM900EV3B,
+// MM900EV-Lite boards.
 // Each of these boards has an FT900 device.
 
 #ifdef __cplusplus
  extern "C" {
 #endif
 
-#define GPIO_UART0_TX 48
-#define GPIO_UART0_RX 49
-#define GPIO_ETH_LED0 61
-#define GPIO_ETH_LED1 62
-#define GPIO_REMOTE_WAKEUP_PIN 18
-#define USBD_VBUS_DTC_PIN 3
+#include <tinyprintf.h>
```

Comment:
"tinyprintf.h" is included but its path ($(TOP)/$(FT9XX_SDK)/3rdparty/tinyprintf" is not added to `INC +=`. I still haven't compile with ft90x yet, please double check to see if path is needed.

Reply:
Yes, indeed. Sorry, that file was used for debugging and was included in recent work on the sdk as you found. It's not required in TinyUSB and I've updated the PR to reflect this and tested against the latest code.

Reply:
no problem at all, this is why including port build in the ci is important. We should try to this in the future. ( I still need to get my PC setup to give this port a try).